### PR TITLE
PYTHON-4496 Add integration test

### PR DESCRIPTION
**Pull request by blink1073**
Depends on https://github.com/mongodb/mongo-python-driver/pull/1677

### CODE REVIEW

**Comment by blink1073** [comment]
evergreen retry

File: .evergreen/config.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+pre:
+  - func: setup
+  - func: bootstrap mongo-orchestration
+
+post:
+  - func: teardown
+  - command: attach.xunit_results
+    params:
+      file:  "mongo-python-driver/xunit-results/TEST-*.xml"
```
Comment (NoahStapp) [inline .evergreen/config.yml:69]: Should this be `$PROJECT_DIRECTORY` instead?

File: .github/reviewers.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# List of reviewers for auto-assignment of reviews.
```
Comment (NoahStapp) [inline .github/reviewers.txt:1]: Is Shane an intentional omission here?

**Review by NoahStapp — CHANGES_REQUESTED** [review]

File: .evergreen/config.yml (hunk 1 above)
Comment (blink1073) [reply .evergreen/config.yml:69]: No, the results are in the separate checkout directory for pymongo

File: .github/reviewers.txt (hunk 2 above)
Comment (blink1073) [reply .github/reviewers.txt:1]: Yes, this matches pymongo

**Review by NoahStapp — APPROVED** [review]

**Pull request merged**